R/expr__expr.R

```diff
@@ -1942,7 +1942,15 @@ Expr_product = "use_extendr_wrapper"
 #' pl$DataFrame(iris)$select(pl$col("Species")$n_unique())
 Expr_n_unique = "use_extendr_wrapper"
 
-
+#'  Approx count unique values
+#' @keywords Expr
+#' @description
+#' This is done using the HyperLogLog++ algorithm for cardinality estimation.
+#' @aliases approx_unique
+#' @return Expr
```

Comment:
```suggestion
#' @return Expr
#' @docType NULL
```

cf bottom of first post here: https://github.com/pola-rs/r-polars/issues/137#issue-1676063640

This needs to be added everywhere the function is just a character `"use_extendr_wrapper"`